astro/src/content/blog/announcing-fusionauth-1-50.mdx

```diff
@@ -0,0 +1,122 @@
+---
+publish_date: 2024-04-24
+title: Announcing FusionAuth 1.50
+description: This update includes OAuth scopes support, a UserInfo lambda, and an easier setup screen.
+authors: Dan Moore
+image: /img/blogs/release-1-50/fusionauth-1-50.png
+categories: Release Updates, Product
+tags: release notes, oauth, scopes, custom scopes, lambdas, userinfo, setup wizard
+excerpt_separator: "{/* more */}"
+---
+FusionAuth version 1.50.0 shipped on April 24, 2024. This version includes OAuth scopes support, a UserInfo lambda, an easier setup screen, and more.
+
+{/* more */}
+
+I hereby dub 1.50.0 the "Scopes Salamander" release.
+
+![I like scopes.](/img/blogs/release-1-50/salamander.png)
+
+All in all, there are 20 issues, enhancements, and bug fixes included in the 1.50.0 release. As always, please see the [release notes](/docs/release-notes/#version-1-50-0) for a full breakdown of the changes between 1.49.2 and 1.50.0, including schema changes.
+
+Let's look at some of the more notable features in this release.
+
+## OAuth Scopes
+
+OAuth scopes let you create coarse grained sets of permissions that you can ask an end user to consent to. 
+
+This isn't super useful if all of the applications delegating authentication and authorization to your identity store are owned by your organization. But if you want to build a platform or ecosystem, where third party developers and organizations access user data held by you, scopes solve that problem elegantly.
```

Comment:
I can't help but ask for a bad pun here. It would require that we flip the sentences though: "If you want to build a platform or ecosystem, where third party developers and organizations access user data held by you, scopes solve that problem elegantly. Conversely, if all of the applications delegating authentication and authorization to your identity store are owned by your organization, that use is...out of scope."

Reply:
I like it! I'm going to wait for a devrel review, but can see this working :)